content/yaml/yaml-conditional-run.md

```diff
@@ -75,21 +65,24 @@ The `condition` you specify will be evaluated during the build. The build will b
 
 The current environment is accessible under the `env` variable.
 
-For example, the build will not run if the current branch is not master:
+For example, the build will continue if it was not triggered by a draft pull request update:
 
 ```yaml
 workflows:
-  build-master:
-    name: Build master branch
+  build:
+    name: Build on PR update
+    triggering:
+      events:
+        - pull_request
     when:
-      condition: env.FCI_BRANCH != 'master'
+      condition: not event.pull_request.draft
 ```
 
 Webhook payload is accessible under the `event` variable. Note that `event` is not available if the build is started manually from the UI or by a schedule.
 
-Be sure to check the webhook event body in your application settings on the Webhooks tab.
+Please check the structure of the webhook payloads that your git provider sends. **Webhooks** tab in application settings contains recently received webhooks.
 
-Note that in addition to `and` it is possible to use other logical operators in conditions, e.g. `not`, `or`.
+Note that in addition to `not` it is possible to use other logical operators in conditions, e.g. `and`, `==`.
```

Comment:
Move this part right after the first sentence in this section and change the emphasis to: "You can use logical operators in conditions, e.g. not, and ...